LondonBritishLibrary/orient/BLorient812.xml

```diff
@@ -49,50 +132,120 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">199</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>311</height>
-                                        <width>270</width>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>12.25</height>
+                                        <width>10.625</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="26 28" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is neatly written , but with many erasures and alternations, of  the earlier part of <date>17th century.</date>
+                                <desc>The manuscript is neatly written ,
+                                  but with many erasures and alternations, of  the earlier part
+                                  of <date>17th century.</date>
                                 </desc>
-                             
+
                                 <date notBefore="1670" notAfter="1700" resp="PRS10171Wright"/>
-                             
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"></locus>
+                              <desc>
+                                A rude table of the signs of Zodiac.
```

Comment:
@smaugustine , can you help us? I always wonder if Wright's "rude" (which probably means what is described as "crude"  for instance in Ethio-SPaRe descriptions) would sound strange to modern English speakers and should be substituted by another word in these descriptions.